page.php

```diff
@@ -21,18 +21,26 @@
  * @since    Timber 0.1
  */
 
+$context['copyright_text'] = get_option( 'copyright', '' ) ? get_option( 'copyright' ) : '';
+
 $context = Timber::get_context();
 $post = new TimberPost();
 $context['post'] = $post;
 
-$page_meta_data = get_post_meta($post->ID);
-$context['header_title'] = $page_meta_data['p4_title'][0] == '' ? $post->title : $page_meta_data['p4_title'][0];
-$context['header_subtitle'] = $page_meta_data['p4_subtitle'][0];
-$context['header_description'] = $page_meta_data['p4_description'][0];
+$page_meta_data = get_post_meta( $post->ID );
+$context['header_title']        = $page_meta_data['p4_title'][0] === '' ? $post->title : $page_meta_data['p4_title'][0];
```

Comment:
There is case where $page_meta_data['p4_title'][0] is null and null === '' is false, so it displays null.
I'd suggest
`! $page_meta_data['p4_title'][0] ? $post->title : $page_meta_data['p4_title'][0];`
@dashaluna @sagarsdeshmukh

Reply:
Actually better to check type as well
`null === $page_meta_data['p4_title'][0] ? $post->title : $page_meta_data['p4_title'][0];`

@dashaluna @sagarsdeshmukh

Reply:
I fixed the above.